Classes/Form/FormDataProvider/TcaCTypeItems.php

```diff
@@ -54,7 +54,7 @@ public function addData(array $result)
             $result['processedTca']['columns'][$field]['config']['items'] = array_filter(
                 $result['processedTca']['columns'][$field]['config']['items'],
                 function ($item) use ($allowedValues) {
-                    return in_array($item['value'] ?? $item[1], $allowedValues);
+                    return in_array($item['value'] ?? $item[1], $allowedValues) || $item['value'] === '--div--';
```

Comment:
Hi @derhansen 

Thank you for this patch. I would love to see the new comparison as first one (as it is the faster comparison) and with the legacy fallback. Would you mind to change it in your pull request?

Reply:
Done